Amethyst/Preferences/UserConfiguration.swift

```diff
@@ -334,8 +337,8 @@ class UserConfiguration: NSObject {
                 alert.messageText = "Error loading configuration"
                 alert.runModal()
             }
-        } else if FileManager.default.fileExists(atPath: amethystJSONConfigPath, isDirectory: nil) {
-            configurationJSON = jsonForConfig(at: amethystJSONConfigPath)
+        } else if FileManager.default.fileExists(atPath: amethystXDGConfigPath, isDirectory: &isDirectory) {
+            configurationJSON = jsonForConfig(at: isDirectory.boolValue ? amethystXDGConfigPath.appending("/amethyst") : amethystJSONConfigPath)
```

Comment:
Should it be `/.amethyst` rather than `/amethyst`?

Reply:
If its in your home directory it usually is .amethyst.yml, but if you follow the XDG, it should be $HOME/.config/amethyst/amethyst.yml

Reply:
Home directory: $HOME/.amethyst.yml

XDG-based: $HOME/.config/amethyst/amethyst.yml

Reply:
Yeah that's how they usually are because files don't need to be hidden if it's not in the home directory.

Some examples on my machine:
```
~/.config/alacritty/alacritty
~/.config/nvim/init.lua
~/.config/git/config
```

One exception I see is [zsh](https://zsh.sourceforge.io/Doc/Release/Files.html) where the hidden file notation is consistently used (but they use `$ZDOTDIR` rather than `$XDG_CONFIG_HOME`):
```bash
> ls -la .config/zsh/
total 120
drwxr-xr-x  6 eric  staff    192 18 Oct 11:01 .
drwxr-xr-x  3 eric  staff     96 12 Oct 11:07 ..
-rw-r--r--  1 eric  staff  47799 12 Oct 11:12 .zcompdump
-rwxr-xr-x@ 1 eric  staff    642 18 Oct 20:40 .zlogin
-rw-r--r--  1 eric  staff     83 18 Oct 10:24 .zprofile
-rw-r--r--  1 eric  staff   1095 18 Oct 11:01 .zshrc
```

@ianyh What do you prefer?

Reply:
Got it. The intention is to meet a standard, and it seems like there's at least a convention, so let's go with not using hidden file name.